packages/lodestar/src/chain/blocks/process.ts

```diff
@@ -107,13 +107,13 @@ export async function updateDepositMerkleTree(
     newState.eth1Data.depositCount - newState.eth1DepositIndex
   );
   const [depositDatas, depositDataRootList] = await Promise.all([
-    db.depositData.getAllBetween(newState.eth1DepositIndex, upperIndex),
+    db.depositData.values({gt: newState.eth1DepositIndex, lt: upperIndex}),
```

Comment:
db.depositData.filter would be more common to use but I don't have strong preference

Reply:
reason for this convention is to expose the key-value nature of `Repository`
`keys` - return [] of keys
`keysStream` return stream of keys
`values` - return [] of values
`valuesStream` return stream of values
`entries` return [] of {key, value}
`entriesStream` return stream of {key,value}

(I'm planning on using `keys` soon in #855)